components/welcome-mat/index.jsx

```diff
@@ -206,6 +206,9 @@ class WelcomeMat extends React.Component {
 				<div
 					className={classNames(
 						'slds-welcome-mat',
+						this.props.variant === 'info-only'
+							? 'slds-welcome-mat_info-only'
+							: null,
```

Comment:
You can use this format instead of ternary operator `{'slds-welcome-mat_info-only' :  this.props.variant === 'info-only'}`

Reply:
sure, that should be compact :) Thank you, I will update it.